pom.xml

```diff
@@ -1,141 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No longer needed. Since we are using gradle.

Reply:
🎉